news/2023/2023-07-13-osu-mania-4k-world-cup-2023-registrations-now-open.md

```diff
@@ -0,0 +1,47 @@
+date: 2023-07-13 12:00:00 +0000
+---
+
+The seasons may change but through it all our love of rice and noodles remains! The osu!mania 4K World Cup is back for its 10th edition!
+
+![](/wiki/Tournaments/MWC/2023_4K/img/mwc4k2023-banner.jpg)
+
+In keeping up with our ongoing efforts to introduce different mechanics for each world cup's format, the osu!mania 4K mappoolers cooked more than just rice and noodles. For the first time ever in a world cup, we are introducing a protect system! Now, you might be asking yourself, what even is a protect? Am I finally going to be able to protect my keyboard from those impossible chordjacks? And you would be wrong... No keyboard will be spared. The main idea behind this system is that each team can exclude *one* beatmap from being banned by either team, effectively "protecting" it. We look forward to seeing how this will change the pick and ban process!
+
+For more details, refer to the [osu!mania 4K World Cup 2023 wiki article](/wiki/Tournaments/MWC/2023_4K). You can discuss this event in the [official discussion thread](https://osu.ppy.sh/community/forums/topics/1792305) or at the public [osu! community server](https://discord.gg/0Vxo9AsejDkGlk3H).
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | Jul 13 – Jul 27 (23:59 UTC) |
+| Qualifier showcase | Aug 06 (14:00 UTC) |
+| Qualifier stage | Aug 12 & Aug 13 |
+| Round of 32 | Aug 19 & Aug 20 |
+| Round of 16 | Aug 26 & Aug 27 |
+| Quarterfinals | Sep 02 & Sep 03 |
+| Semifinals | Sep 09 & Sep 10 |
+| Finals | Sep 16 & Sep 17 |
+| Grand Finals | Sep 23 & Sep 24 |
+
+## Prizes
+
+The osu!mania 4K World Cup 2023 offers a $2,000 prize pool, along with limited-edition merch!
+
+- **1st:** 50% of the prize pool, single-run merch, profile badge, **osu!mania 4K Champion** user title for one year
+- **2nd:** 30% of the prize pool, single-run merch, profile badge
+- **3rd:** 20% of the prize pool, single-run merch, profile badge
+
+## How do I register?
+
+- **[Click here!](https://osu.ppy.sh/community/tournaments/39)**
+- Teams will be formed by each country's captain *after* the registration phase has concluded.
+- **The deadline for registering is July 27, 23:59 UTC. Make sure to submit yours before then!**
+
+*Notice: you may only participate if your position in the osu!mania 4K global ranking is #5000 or higher, and you have not violated the [osu! community rules](/wiki/Rules) in the past 12 months.*
+
+If your registration passes our internal checks, you will be placed on the candidate list for your country. When that country's captain has been assigned, the list will be sent to them. This may take a while, so please be patient.
+
+*Remember that any registrations submitted after the deadline will not be considered under any circumstances — submit your registration ahead of time to avoid any last-minute headaches. If you run into any issues, send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).*
```

Comment:
signoff is required

```suggestion
*Remember that any registrations submitted after the deadline will not be considered under any circumstances — submit your registration ahead of time to avoid any last-minute headaches. If you run into any issues, send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).*

—LeoFLT
```